client/packages/common/src/intl/locales/en/common.json

```diff
@@ -89,6 +89,7 @@
   "label.customer-soh": "Their SOH",
   "label.tax": "Tax",
   "label.date": "Date",
+  "label.date-of-birth": "D.O.B",
```

Comment:
I thought it might be too short, and looks silly on my monitor, but thought to leave the little version in order to cope with smaller screen sizes. Will revise that, thanks.